app/logic/fileHandler.py

```diff
@@ -94,3 +94,9 @@ def deleteFile(self, fileId):
         if not AttachmentUpload.select().where(AttachmentUpload.fileName == file.fileName).exists():
             path = os.path.join(self.path, file.fileName)
             os.remove(path)
+    
+    def displayFile(self, fileId):
```

Comment:
This is a rather small nitpick but it might be good to modify the function name. 

Sure, we call this function when selecting a file to be displayed but, the function also removes a file from display in the case where one is already being displayed and we change to a different file.